src/sonic-yang-models/tests/yang_model_pytests/conftest.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import yang as ly
+from json import dumps
+from glob import glob
+
+
+class YangModel:
+
+    model_dir = './yang-models'
```

Comment:
>. [](http://example.com/codeflow?start=17&length=1)

Relative path means you assume the current folder must be some specific one. Could you not to assume the current folder?

Reply:
done